rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/CompactView.java

```diff
@@ -687,7 +548,7 @@ private void highlightGroup(boolean selected) {
 
         @Override
         public boolean equals(Object o) {
-            NodeTile t = (NodeTile) o;
+            Tile t = (Tile) o;
             if (this.node != null && t.node != null) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') A "NullPointerException" could be thrown; "t" is nullable here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2259)